src/pombot/commands/start.js

```diff
@@ -1,11 +1,29 @@
 /*
- * A stub for the "start" command.
+ * The `start` command, which starts a Pom.
  */
 import {createCommand} from 'chatter';
+import states from '../states';
 
-export default createCommand({
-  name: 'start',
-  description: 'Starts a pom timer.',
-}, () => {
-  return `> Theoretically I should be starting a pom right now.`;
-});
+export default function(pom) {
+  return createCommand({
+    name: 'start',
+    description: 'Starts a pom timer.',
+  }, message => {
+
+    switch (pom.state) {
+      case states.RUNNING:
+        // if pom is running, stay minutes left in current pom
+        return `there is already a pom running, there are *${pom.getMinutes(pom.timeLeft)} minutes* left.`;
+        break;
+      case states.ON_BREAK:
+        // if pom is on break, say minutes left in break
```

Comment:
do we need this comment? i think you've composed this nicely enough that just reading the code itself says this comment